packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +815,381 @@ components:
+              codec:
+                type: string
+                description: Codec of the track
+                example: aac
+        profiles:
+          type: object
+          additionalProperties: false
+          description: HLS Profile manifests associated with the asset
+          properties:
+            kind:
+              type: string
+              description: Type of the profile
+              example: manifest
+            name:
+              type: string
+              description: Filename of the manifest
+              example: variant_1080.m3u8
+            width:
+              type: number
+              description: width of the profile
+              example: 1920
+            height:
+              type: number
+              description: height of the profile
+              example: 1080
+            fps:
+              type: number
+              description: Frame rate of the profile
+              example: 30
+            id:
+              type: string
+              description: ID of the profile
+              example: ABC123
+            duration:
+              type: number
+              description: Duration of the profile in seconds
+              example: 23.8238
+            bitrate:
+              type: number
+              description: Bitrate of the profile in bits per second
+              example: 1000000
+        playbackUrl:
+          type: string
+          description: Playback url of the playlist manifest
+          example: https://cdn.livepeer.com/ABC123/playlist.m3u8
+        status:
+          type: string
+          description: Status of the asset
+          example: ready
+        originTaskId:
+          type: string
+          description: Task ID of the task that created the asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        relatedTasks:
+          type: array
+          description: List of task IDs associated with the asset
+          items:
+            $ref: "#/components/schemas/task"
+        sourceAssetId:
+          type: string
+          description:
+            ID of the source asset (root) - If missing, this is a root asset
+          example: 09F8B46C-61A0-4254-9875-F71F4C605BC7
+        nft:
+          type: array
+          description: List of NFTs minted with the asset as a source
+          items:
+            type: object
+            additionalProperties: false
+            properties:
+              chain:
+                type: string
+                description: Chain of the NFT
+                example: eth
+              contract:
+                type: string
+                description: Contract of the NFT
+                example: 0x0000000000000000000000000000000000000000
+              token:
+                type: number
+                description: Token id of the NFT
+                example: 123
+
+    task:
+      type: object
+      table: task
+      additionalProperties: false
+      properties:
```

Comment:
I think some fields are missing:
 - An indexed `sourceAssetId`, so that we can always search for tasks related to a given task. It would work differently from the `sourceAssetId` of the assets themselves tho, pointing to the direct parent and not to the tree root, so we might want to call it something else. I think `parentAssetId` could make sense then. It can also be `null` in case of "import" tasks for example. In the future it might even be a list of assets, in case of stitching 2 videos together for example, but for now I think we're good with a single parent ID, which will be simpler (it's kinda complex to get indexes on JSON arrays as well (tho definitely possible))
 - The output of the task. I think on our design we had thought of a `status` **object** instead of just a string, and inside the status object we would have both the current "phase" (k8s-inspired) which is that `Pending`/`Running` etc, a timestamp of the last change, and then the `output` of the task after it's down (which should follow the same pattern as `params` IMO, with the separate sub-fields for each task type). So I think the current `status` and `updatedAt` should be moved to an actual `status` object (`status`=>`phase`) and then we add the `output` there as well. WDYT?